doc/articles/get-started-licensing.md

```diff
@@ -0,0 +1,63 @@
+---
+uid: Uno.GetStarted.Licensing
+---
+
+# Sign in with Uno Platform
```

Comment:
@sasakrsmanovic, @jeromelaban I think we could use this as well in the TOC as a header in the troubleshoot section of our doc.

And again should we refer to this troubleshoot doc somewhere in the "Create an app" section of our doc ?

Reply:
@morning4coffe-dev let's use "Sign in with Uno Platform" in the TOC for now in the troubleshoot section